### DISCUSSION TITLE
Is it possible to get the thumbnail of the video?

### DISCUSSION THREAD

**Discussion by Nex2944** [Androz2091/discord-player#1057] [General]
Hello i was wondering if its possible to get the thumbnail of a video from the result?
Also <result>.title shows undefined

**Comment by twlite** [comment]
yes `<Track>.thumbnail` https://github.com/Androz2091/discord-player/blob/c7a6bf2ec106ddfc3a567712f4683456419269f5/src/Structures/Track.ts#L13

**Reply by Nex2944** [reply]
![image](https://user-images.githubusercontent.com/72197572/155362813-ef01a928-e620-4f29-b648-d2b60784b7e2.png)
![image](https://user-images.githubusercontent.com/72197572/155362852-135f5caf-61b6-4c69-bd27-9a402dbfaf27.png)

The thumbnail dosent show up 🤔
